README.rst

```diff
@@ -112,6 +112,8 @@ Below is the full listing of options::
                             this only triggers if there is only one star import in
                             the file; this is skipped if there are any uses of
                             `__all__` or `del` in the file
+      --populate-all        populate `__all__` with unused import found in the
```

Comment:
Too generic.  `--populate-module-dunder-all` ?

Reply:
Yeah, but... `dunder`?

Reply:
Oh I think you mean `under`

Reply:
I think [dunder](https://wiki.python.org/moin/DunderAlias) is good.